### PR TITLE
updates to site design

### PR DESCRIPTION
**Pull request by alexcontini**
@mandy-chessell 

This one should be better. I have made sure that all the .md files are not changed at all, and linked all the images to the actual folder and file rather than using hard-coded links. 

The only issue I am having is that I had to make changes to the theme directly in github, and that adds a commit automatically that I can't sign. Let me know if there are any issues with that

### CODE REVIEW

**Comment by planetf1** [comment]
Just noticed the build failed
 a) the DCO signoff is needed
 b) The RAT (license check) failed - the SPDX identifiers are missing in some html ..

File: _layouts/homepage.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```
Comment (cmgrote) [inline _layouts/homepage.html]:
```suggestion
<!-- SPDX-License-Identifier: Apache-2.0 -->
<!-- Copyright Contributors to the ODPi Egeria project. -->
<!DOCTYPE html>
```

File: assets/css/style-sub.scss [hunk 2]
```diff
@@ -0,0 +1,261 @@
+---
```
Comment (cmgrote) [inline assets/css/style-sub.scss]:
```suggestion
/* SPDX-License-Identifier: Apache-2.0
    Copyright Contributors to the ODPi Egeria project. */
---
```

File: assets/css/style.scss [hunk 3]
```diff
@@ -0,0 +1,292 @@
+---
```
Comment (cmgrote) [inline assets/css/style.scss]:
```suggestion
/* SPDX-License-Identifier: Apache-2.0
    Copyright Contributors to the ODPi Egeria project. */
---
```

**Pull request closed without merging**